boards/arm/nucleo_wb55rg/nucleo_wb55rg.yaml

```diff
@@ -19,3 +19,4 @@ supported:
   - arduino_gpio
   - arduino_i2c
   - arduino_spi
+  - usb device
```

Comment:
It seems you copied this from here:
https://github.com/zephyrproject-rtos/zephyr/blob/628a0336bf0e7fae6dfb9f29fcae43063602d95e/boards/arm/nucleo_l4r5zi/nucleo_l4r5zi.yaml#L17

This is incorrect and should be changed to `usb_device`.

Reply:
@stephanosio I did indeed. Fixed. Thanks!

Reply:
@petejohanson, seems you didn't pushed the fix

Reply:
@erwango Argh. I'm failing at git this morning. Amended the commit and force pushed w/ fixes (for real this time).